.travis.yml

```diff
@@ -11,9 +11,7 @@ matrix:
 script:
   - yarn run lint
   - yarn run test:prebuild
-  - export HASH=$(cksum dist/index.js)
   - yarn run build
-  - bash -c 'if [[ "$(cksum dist/index.js)" != $HASH ]]; then exit 1; fi'
```

Comment:
This test is here to ensure I build the lib before pushing.
Why do you need to remove it ?

Reply:
In other projects i contribute, its requested to not not commit the updated build file in PR. It does only in release.

If you want i can commit the build file in the PR and leave this check

Reply:
idk I took this habbit here, what is better ?